scripts/sut-tests.sh

```diff
@@ -102,7 +102,8 @@ fi
 
 echo '🚩 Install local DCG'
 composer -d"$DRUPAL_DIR" config repositories.dcg "$(printf '{"type": "path", "url": "%s", "options": {"symlink": false}}' $SELF_DIR)"
-composer -d$DRUPAL_DIR require chi-teck/drupal-code-generator:3.x
+composer -d$DRUPAL_DIR info --available chi-teck/drupal-code-generator
```

Comment:
The info command is not required. It was just to see that the correct version was loaded. Should I remove this?

Reply:
I removed the version in ``composer require`` because the branch name is different for Pull Requests. git is in `detached head` state during CI for Pull Requests.

Reply:
I was asking about `info` command. Will this be better to use `composer show -- chi-teck/drupal-code-generator` instead? Seems this is an intended way to get the installed version of a package.

Reply:
Well, `composer info` is just an alias for `composer show`. I've removed `--available` option to see the only installed version.